calculator/src/components/Activities.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default class Navbar extends React.Component {
+
+  render() {
+      console.log(this.props.activities)
+      const activities = this.props.activities;
+
+    return (
+      <div>
+        <h1>Look at your day!</h1>
+        {activities.sleep != '' && activities ? 
+        <ul className='activities'>
+          <li className='sleepBasis'>{activities.sleep}</li>
```

Comment:
Not sure if you ran out of time but it would be nice to have these displaying in multiple rows instead of all on one row.